ui/src/main/js/view/templates/run-changeset.hbs

```diff
@@ -1,9 +1,13 @@
 <div class="alert alert-info run-changeset-details {{kind}}-repo">
+    {{#if contributors.0.authorJenkinsId}}
     <div class="header">{{commitCount}} commit{{#ifCond commitCount '>' 1}}s{{/ifCond}} by <span class="contributor">{{contributors.0.authorJenkinsId}}</span>{{#ifCond contributorCount '>' 1}} and others{{/ifCond}}</div>
```

Comment:
You're right, it's a bit simpler. In this case, don't think it's quite a big enough change to run through another review cycle on this, but I reserve the right to go back and make the change later.

Worth noting: that 3rd line would be better written as #if not #ifCond for handlebars.